upgrade/capsule.py

```diff
@@ -135,6 +136,9 @@ def satellite_capsule_upgrade(cap_host, sat_host):
 
     if settings.upgrade.foreman_maintain_capsule_upgrade:
         foreman_maintain_package_update()
+        # pulp2 removal required for 7.0 ystream upgrade
```

Comment:
It would be good if you mention the Bugzilla number in this comment.

Reply:
True. I'm going to add BZ#

Reply:
BZ # added.